src/map/skill.cpp

```diff
@@ -54,6 +54,8 @@ using namespace rathena;
 static struct eri *skill_timer_ers = NULL; //For handling skill_timerskills [Skotlex]
 static DBMap* bowling_db = NULL; // int mob_id -> struct mob_data*
 
+bool damage_flag = false; //Night Watch Skill
```

Comment:
Should be checked if really necessary.
I rather dislike adding a global variable for something like this.